src/db.js

```diff
@@ -159,17 +159,18 @@ export async function dbSave(event) {
 
 // db queries
 // ~
-export async function dbGetHomeFeedNotes(
-  limit = 50,
-  since = Math.round(Date.now() / 1000)
-) {
+export async function dbGetHomeFeedNotes(limit, since) {
   let result = await db.query('main/homefeed', {
     include_docs: true,
     descending: true,
     limit,
     startkey: since
   })
-  return result.rows.map(r => r.doc)
+
+  //Get doc-uments from pouchdb result
+  let events = result.rows.map(r => r.doc)
+
+  return events
 }
```

Comment:
this changes nothing, right?

I changed this entire file to run pouchdb on a webworker on master, so everything you do here will conflict. but since this change isn't really a change I think we can just discard it.